src/index.js

```diff
@@ -74,10 +77,14 @@ export default function ({ types: t, traverse: _traverse }) {
         const { parentPath } = path;
         const explicitReturn = parentPath.isReturnStatement();
         const implicitReturn = parentPath.isArrowFunctionExpression();
-
-        const openingElement = elementOpenCall(path.get("openingElement"), this);
-        const closingElement = elementCloseCall(path.get("openingElement"), this);
-        const children = buildChildren(path.get("children"), this);
+        const openingElementPath = path.get("openingElement");
+
+        const openingElement = elementOpenCall(openingElementPath, this);
+        const closingElement = elementCloseCall(openingElementPath, this);
+        const isSkipping = hasSkip(openingElementPath.get("attributes"), this) ||
+          isComponent(openingElementPath.get("name"), this);
```

Comment:
I don't use custom components, but if I were to use them I too would probably find this auto-skipping a bit surprising. I agree with @mairatma and personally I'd prefer to only skip elements that have the tag, be they components or not (we can use skip tag with components, can we not?), but again I don't use the components so I'm ok with whatever you decide.